### PR TITLE
1282 feature request ktoast add do not show again option

### PR DESCRIPTION
**Pull request by Wagnerp**
* Start working on #1282
   - Note: Setting the 'checked' property on the check box on initialisation does not currently work, whereas `CheckState` does work.

image

### CODE REVIEW

File: Source/Krypton Components/Krypton.Toolkit/Controls Visuals/Notification Toast/Basic/Left to Right/VisualToastNotificationBasicForm.cs [hunk 1]
```diff
@@ -212,6 +214,20 @@ private void VisualToastNotificationBasicForm_Load(object sender, EventArgs e)
             _soundPlayer?.Play();
         }
 
+        private void UpdateSizing()
+        {
+            // Add some height, if form border style equals 'None'
+
+            var width = Size.Width;
```
Comment (Smurf-IV) [inline Source/Krypton Components/Krypton.Toolkit/Controls Visuals/Notification Toast/Basic/Left to Right/VisualToastNotificationBasicForm.cs:221]: These are only needed inside the if statement.

File: Source/Krypton Components/Krypton.Toolkit/Controls Visuals/Notification Toast/Basic/Left to Right/VisualToastNotificationBasicForm.cs [hunk 2]
```diff
@@ -390,6 +406,8 @@ protected override void OnLoad(EventArgs e)
                 Hide();
             }
 
+
```
Comment (Smurf-IV) [inline Source/Krypton Components/Krypton.Toolkit/Controls Visuals/Notification Toast/Basic/Left to Right/VisualToastNotificationBasicForm.cs:409]: Why extra lines ?

File: Source/Krypton Components/Krypton.Toolkit/General/GlobalStaticValues.cs [hunk 3]
```diff
@@ -30,6 +30,8 @@ public class GlobalStaticValues
         /// <summary>The global button padding</summary>
         public const int GLOBAL_BUTTON_PADDING = 10;
 
+        internal const int DEFAULT_PADDING = 10;
```
Comment (Smurf-IV) [inline Source/Krypton Components/Krypton.Toolkit/General/GlobalStaticValues.cs:33]: Does this need to be scaled for different screen settings ?

**Review by Smurf-IV — CHANGES_REQUESTED** [review]

File: Source/Krypton Components/Krypton.Toolkit/Controls Visuals/Notification Toast/Basic/Left to Right/VisualToastNotificationBasicForm.cs (hunk 1 above)
Comment (Wagnerp) [reply Source/Krypton Components/Krypton.Toolkit/Controls Visuals/Notification Toast/Basic/Left to Right/VisualToastNotificationBasicForm.cs:221]: Now inside the `if` statement

File: Source/Krypton Components/Krypton.Toolkit/Controls Visuals/Notification Toast/Basic/Left to Right/VisualToastNotificationBasicForm.cs (hunk 2 above)
Comment (Wagnerp) [reply Source/Krypton Components/Krypton.Toolkit/Controls Visuals/Notification Toast/Basic/Left to Right/VisualToastNotificationBasicForm.cs:409]: Removed extra lines. I think they were left over from some testing.

File: Source/Krypton Components/Krypton.Toolkit/General/GlobalStaticValues.cs (hunk 3 above)
Comment (Wagnerp) [reply Source/Krypton Components/Krypton.Toolkit/General/GlobalStaticValues.cs:33]: Yes, I've left a todo note in the affected `if` statement

**Pull request merged**